docs/swagger/spec.yml

```diff
@@ -611,38 +655,225 @@ paths:
           description: Thumbnail UUID was not found
           schema:
             $ref: '#/definitions/Error'
-  /boundaries/:
+  /tokens/:
     get:
-      summary: Paginated list of boundaries
+      summary: |
+        Retrieve a list of refresh token summaries. Note: Only names and identifiers are revealed,
+        refresh tokens themselves are only revealed once, when created for security purposes.
       tags:
-        - Imagery
+        - Authentication
+      responses:
+        200:
+          description: List of refresh token identifiers
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/RefreshToken'
+        404:
+          description: No refresh tokens found for user
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: |
+        Request a new authorization token to use in requests
```

Comment:
This endpoint uses a refresh token to generate a new auth token. refresh tokens are generated on the frontend using auth0 and are never seen by our backend.  We don't actually store any information about the refresh tokens on our server at all, and pass requests onto auth0 using the management API for actions such as listing / deleting refresh tokens.